tests/kernel/pipe/pipe_api/src/test_pipe_contexts.c

```diff
@@ -142,7 +142,9 @@ static void tpipe_put_small_size(struct k_pipe *ppipe, k_timeout_t timeout)
 	for (int i = 0; i < PIPE_LEN; i += wt_byte) {
 		/**TESTPOINT: pipe put*/
 		to_wt = 15;
-		k_pipe_put(ppipe, &data[i], to_wt, &wt_byte, 1, timeout);
```

Comment:
`zassert_false` evaluates a boolean operation but `k_pipe_put` return an integer. Although C interprets anything != 0 as true it is not semantically correct IMHO. Would be better if you use `zassert_equal` here.